packages/seal/src/client.ts

```diff
@@ -45,10 +45,12 @@ export interface SealClientExtensionOptions {
 
 export interface SealClientOptions extends SealClientExtensionOptions {
 	suiClient: SealCompatibleClient;
+	apiKey?: string;
 }
 
 export class SealClient {
 	#suiClient: SealCompatibleClient;
+	#apiKey?: string;
```

Comment:
it should be per key server, and let the user define to header name.
perhaps let's replace serverObjectIds with something like KeyServerConfig[] where
class KeyServerConfig { objectId, weight, ?apiKeyHeader, ?apiKey }